src/rpcserver.cpp

```diff
@@ -27,8 +27,6 @@
 #include <boost/signals2/signal.hpp>
 #include <boost/thread.hpp>
 #include <boost/algorithm/string/case_conv.hpp> // for to_upper()
-#include <boost/algorithm/string/classification.hpp>
-#include <boost/algorithm/string/split.hpp>
```

Comment:
Move this into the first commit, or into an intermediate commit (so the two together correspond to upstream's commit, and the final commit is our further changes).

Reply:
Done